stories/control-group.stories.js

```diff
@@ -0,0 +1,251 @@
+import React from 'react'
```

Comment:
Add story that pass `height` prop to controls.

Reply:
done